docs/api/python/index.rst

```diff
@@ -29,10 +29,18 @@ Python API
    tir
    te
    driver
+   relay/index
```

Comment:
There is actually also a "memory" subfolder under relay/op, we probably need to add it here as well?

Reply:
I am not too sure for now about where should they go, given these ops are more developer facing(rather than user facing). Perhaps we can skip it(as it wasn't in the original doc) and then dicscuss how  to expose them.

`relay.memory` seems can cause confusion in quite some places

Reply:
if we want to add it, probably it should be under relay.op.memory(given it is not user facing)

Reply:
yeah, I agree with this. Let's skip them for now.